index.js

```diff
@@ -27,16 +27,36 @@
     hexBytes[i] = (i + 0x100).toString(16).substr(1);
   }
 
-  // Node & Browser support
   var _crypto;
-  if(typeof crypto !== 'undefined') {
-    _crypto = crypto;
+  setEnvironment();
+  function setEnvironment() {
+    if(isNodeEnvironment) {
+      _crypto = _crypto || require('crypto');
+      module.exports = uuid;
+      return;
+    }
+    else if(isBrowserEnvironment) {
```

Comment:
Looks like a bug here. You don't invoke the function.

Reply:
![](https://www.chronicle.com/blogs/linguafranca/files/2017/11/Nothing-to-See-15a34a2fc727c8.jpg)